source/_includes/footer.liquid

```diff
@@ -6,17 +6,12 @@
   <div class="sl-l-container sl-l-section">
     <div class="sl-l-grid sl-l-grid--full sl-l-large-grid--fit sl-l-large-grid--center sl-l-large-grid--gutters">
       <div class="sl-l-grid__column">
-        Sass &copy; 2006&ndash;{{ 'now' | date: '%Y' }} the Sass team, and numerous contributors.
-        It is available for use and modification under the <a href="https://github.com/sass/dart-sass/blob/master/LICENSE">MIT&nbsp;License</a>.
+        {% renderTemplate 'liquid,md' %}
+          Sass &copy; 2006&ndash;{{ 'now' | date: '%Y' }} the Sass team, and numerous contributors.
+          It is available for use and modification under the [MIT&nbsp;License](https://github.com/sass/dart-sass/blob/master/LICENSE).
```

Comment:
did we want this &nbsp; character here @jgerigmeyer? I'm guessing we are using this to prevent a break but I think I saw this site using widont now?

Reply:
Hm, yeah -- the default markdown parser does not add `widont` or `caps` or anything, but our custom `markdown` filter/shortcode does... I think we could rewrite it as a simple markdown plugin to make that more consistent... 👀